.semaphore/semaphore.yml

```diff
@@ -0,0 +1,21 @@
+version: v1.0
+name: Python
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+  - name: Functions tests
+    task:
+      jobs:
+        - name: main_test.py
+          commands:
+            - sem-version python 3.7
+            - checkout
+            - python -m pip install --upgrade pip
+            - cd Functions/Backend
+            - pip install -r requirements.txt
+            - pip install flake8
+            - 'flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics'
+            - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics

```

Comment:
Looks like it's using style guide enforcement through [flake8](https://flake8.pycqa.org/en/latest/)